content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -268,32 +278,36 @@ once bound to a node, a Pod will never be rebound to another node.
 
 ## Pod lifetime
 
-In general, Pods remain until a human or controller process explicitly removes them.
-The control plane cleans up terminated Pods (with a phase of `Succeeded` or
+In general, Pods remain until a human or
+{{< glossary_tooltip term_id="controller" text="controller" >}} process
+explicitly removes them.
+The control plane cleans up terminated Pods (with a phase of `Succeeded` or 
 `Failed`), when the number of Pods exceeds the configured threshold
 (determined by `terminated-pod-gc-threshold` in the kube-controller-manager).
 This avoids a resource leak as Pods are created and terminated over time.
 
-Three types of controllers are available:
+There are different kinds of resources for creating Pods:
 
-- Use a [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) for Pods that are expected to terminate,
+- Use a {{< glossary_tooltip term_id="deployment" >}},
+  {{< glossary_tooltip term_id="replica-set" >}} or {{< glossary_tooltip term_id="statefulset" >}}
+  for Pods that are not expected to terminate, for example, web servers.
+
+- Use a {{< glossary_tooltip term_id="job" >}}
+  for Pods that are expected to terminate once their work is complete;
   for example, batch computations. Jobs are appropriate only for Pods with
   `restartPolicy` equal to OnFailure or Never.
 
-- Use a [ReplicationController](/docs/concepts/workloads/controllers/replicationcontroller/),
-  [ReplicaSet](/docs/concepts/workloads/controllers/replicaset/), or
-  [Deployment](/docs/concepts/workloads/controllers/deployment/)
-  for Pods that are not expected to terminate, for example, web servers.
-  ReplicationControllers are appropriate only for Pods with a `restartPolicy` of
-  Always.
+- Use a {{< glossary_tooltip term_id="daemonset" >}}
+  for Pods that need to run one per eligible node, because they provide a
+  machine-specific system service.
```

Comment:
Suggest: "For a pod that provides a machine-specific purpose, it may be desirable to have one copy of the pod running on each eligible node in a cluster. In this case, use a DaemonSet to configure the cluster to run the pod on each of the eligible nodes." Or just remove the because clause, the because reason is not necessary to the use of daemonset. It's confusing to read.

Reply:
resolved.